examples/with-redux-toolkit/store.js

```diff
@@ -1,26 +1,12 @@
-import { configureStore, createReducer } from '@reduxjs/toolkit'
+import { configureStore } from '@reduxjs/toolkit';
 
-const initialState = {
-  light: false,
-  lastUpdate: 0,
-  count: 0,
-}
+import clock from './lib/slices/clockSlice';
```

Comment:
Nitpick: I'd prefer to see these imported as `clockReducer` and `countReducer`, and passed to `configureStore` as `count: countReducer`, just to make it clear what's happening.

Reply:
Done at 1cddd52